dev-packages/e2e-tests/test-applications/vue-3/package.json

```diff
@@ -16,6 +16,7 @@
   },
   "dependencies": {
     "@sentry/vue": "latest || *",
+    "pinia": "^2.2.3",
```

Comment:
Should we add pinia as a peer dependency as well with major v2?

Reply:
Sure, added `pinia` as an optional peer dependency. 👍